runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightConfig.java

```diff
@@ -52,4 +52,15 @@ default boolean chatboxTooltip()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 2,
+			keyName = "rightclickoptionTooltip",
+			name = "Right Click Option Tooltips",
+			description = "Whether or not tooltips are shown for options that right-click only."
+	)
+	default boolean isRightClickTooltipEnabled()
+	{
+		return false;
```

Comment:
This should be enabled by default

Reply:
Enabled in commit 32d87ecb832d1494e6ab2146f2beb2d524403623